### PR TITLE
Copy paste

### PR DESCRIPTION
**Pull request by mebster**

### CODE REVIEW

**Review by 0xbase12 — COMMENTED** [review]
This pull-request is a wip or I should do a review? Because I see debug messages in the code I'm not sure

File: code/src/cljs/sixsq/nuvla/ui/utils/semantic_ui_extensions.cljs [hunk 1]
```diff
@@ -77,7 +78,7 @@
   [{:keys [header icon]}]
   [ui/ModalHeader
    (when icon
-     [ui/Icon {:name "add"}])
+     [ui/Icon {:name icon}])
```
[the remark this answers is not in the document]
Comment (mebster) [reply code/src/cljs/sixsq/nuvla/ui/utils/semantic_ui_extensions.cljs:81]: If no icon is provided, `when` will not return anything and there won't be an icon.  But yes we can add the `+` as a default icon as before.

File: code/src/cljs/sixsq/nuvla/ui/utils/semantic_ui_extensions.cljs [hunk 2]
```diff
@@ -35,7 +36,7 @@
   "Provides a menu item that reuses the name for the :name property and as the
    MenuItem label. The optional icon-name specifies the icon to use. The
    loading? parameter specifies if the icon should be spinning."
-  [{:keys [name icon loading?] :as options}]
+  [{:keys [name icon loading? disabled] :as options}]
```
[the remark this answers is not in the document]
Comment (mebster) [reply code/src/cljs/sixsq/nuvla/ui/utils/semantic_ui_extensions.cljs:39]: Removed

File: code/src/cljs/sixsq/nuvla/ui/utils/forms.cljs [hunk 3]
```diff
@@ -20,6 +20,13 @@
   (mapv (fn [{:keys [id label]}] {:value id, :text (or label id)}) descriptions))
 
 
+(def red "#9f3a38")
```
[the remark this answers is not in the document]
Comment (mebster) [reply code/src/cljs/sixsq/nuvla/ui/utils/forms.cljs:23]: I know ;-)... now called `dark-red`

File: code/src/cljs/sixsq/nuvla/ui/apps/views_detail.cljs [hunk 4]
```diff
@@ -249,6 +264,51 @@
...
 
+(defn paste-modal
+  []
+  (let [tr              (subscribe [::i18n-subs/tr])
+        visible?        (subscribe [::subs/paste-modal-visible?])
+        copy-module     (subscribe [::subs/copy-module])
+        module-name     (:name @copy-module)
+        new-module-name (r/atom module-name)
+        form-valid?     (r/atom true)]
+    @copy-module
```
[the remark this answers is not in the document]
Comment (mebster) [reply code/src/cljs/sixsq/nuvla/ui/apps/views_detail.cljs:275]: 👍

File: code/src/cljs/sixsq/nuvla/ui/apps/views_detail.cljs [hunk 5]
```diff
@@ -249,6 +264,49 @@
...
+         [ui/ModalContent
+          [:div (@tr [:paste-modal-content])]
+          [:h5 {:style {:margin-top "10px"}} (:parent-path @copy-module) "/" (:name @copy-module)]
+          [ui/Table {:compact    true
+                     :definition true}
+           [ui/TableBody
+
+            [uix/TableRowField (@tr [:new-name]), :key "new-name", :editable? true,
+             :spec ::spec/name, :validate-form? true, :required? true,
+             :default-value @new-module-name, :on-change #(do (log/error ">" % "<") (reset! new-module-name %)
```
Comment (0xbase12) [inline code/src/cljs/sixsq/nuvla/ui/apps/views_detail.cljs:297]: logging to be removed

**Pull request merged**